lib/OpenQA/Script/CloneJob.pm

```diff
@@ -39,7 +39,7 @@ sub clone_job_apply_settings ($argv, $depth, $settings, $options) {
 
     for my $arg (@$argv) {
         # split arg into key and value
-        unless ($arg =~ /([A-Z0-9_]+)(:(\w+))?(\+)?=(.*)/) {
+        unless ($arg =~ /([A-Z0-9_]+)(:(\w.+?))?(\+)?=(.*)/) {
```

Comment:
I'm wondering if we should really allow any character. I actually don't know if we have any restrictions on test names, would be good to check.
```suggestion
        unless ($arg =~ /([A-Z0-9_]+)(:([\w-]+?))?(\+)?=(.*)/) {
```

Reply:
let me try  the suggested change.